Sources/ArcGISToolkit/Components/Augmented Reality/CalibrationView.swift

```diff
@@ -58,6 +62,7 @@ extension WorldScaleGeoTrackingSceneView {
                                 .font(.body.smallCaps())
                                 .foregroundStyle(.secondary)
                             Spacer()
+                            Text(heading.isLess(than: 0) || heading.isZero ? "" : "+") +
```

Comment:
isn't this the heading delta?

Reply:
Also, the heading control allows you to go over 360 and less than -360. It should limit your calibration to -180...180.

Reply:
> isn't this the heading delta?

Yes. The heading is relative to zero so a separate value is not needed like for the elevation delta which is relative to a starting elevation offset. The property could be renamed.